community/graph-algo/src/main/java/org/neo4j/graphalgo/impl/util/DijkstraBranchCollisionDetector.java

```diff
@@ -40,7 +41,7 @@
 
     public DijkstraBranchCollisionDetector( Evaluator evaluator,
             CostEvaluator costEvaluator,
-            MutableDouble shortestSoFar, double epsilon, Predicate<Path> pathPredicate )
+            org.apache.commons.lang3.mutable.MutableDouble shortestSoFar, double epsilon, Predicate<Path> pathPredicate )
```

Comment:
I'm super confident that we use `org.apache.commons.lang3.mutable.MutableDouble` now.